website/blog/2024-02-20-release-2024-2-is-here/item.md

```diff
@@ -0,0 +1,102 @@
+image: 2024.2-release-blog.png
+---
+
+> **_authentik is an open source Identity Provider that unifies your identity needs into a single platform, replacing Okta, Active Directory, and auth0. Authentik Security is a [public benefit company](https://github.com/OpenCoreVentures/ocv-public-benefit-company/blob/main/ocv-public-benefit-company-charter.md) building on top of the open source project._**
+
+---
+
+We are happy to announce that 2024 is starting off great, with our first release of the year chock full of new features. Take a look at the new features and functionality included in the release, check out the [Release Notes](https://goauthentik.io/docs/releases/2024.2) for more details and upgrade instructions, and enjoy the new features!
+
+![graphic of release highlights](./2024.2-release-blog.png)
+
+<!--truncate-->
+
+We confess we are possibly the most excited about this release than any in a while, with some new Admin-level capabilities, enhanced functionality for developers (our DX game is heating up!), and some great UX/usability and customization enhancements.
+
+Let’s start with some of the big features, the ones that kept us busy over the holidays and into the new year.
+
+### **Remote Access Control** (Enterprise)
+
+The Remote Access Control provider allows you to remotely connect to other machines over RDP, SSH and VNC through authentik. With the RAC provider, we kept implementation consistent with our other providers; you use the same policy engine and customization options that are possible with other providers and use the same user and admin interface.
+
+Typical use cases for remote access include being able to use authentik exactly as you would on your local machine, be that as a _user_ (I need to reset my password, or authenticate to use a specific application that is running on that remote machine), or as an Admin (I need to manage users, create new login flows, etc.).
+
+### Create new apps (and providers!) with an easy Wizard (Preview)
+
+We are very happy to announce our new App Wizard. It's still in Preview, so please send us your feedback!
+
+Gone are the days when you started to create a new application, then realized you needed to first go create a provider (a means of authentication against a remote application), then back to the app to finish up… now the steps for creating both a new app and a new provider are managed by the wizard; easy-peasy. We know this was a too-many-hoops hurdle for some new users, so our new App wizard is a new feature that we are particularly pleased to release. Be sure to check out our [YouTube video](https://www.youtube.com/watch?v=broUAWrIWDI) covering the wizard!
+
+### Audit logging now with field-level changes (Enterprise)
+
+Audit all changes made to models, including which fields were changed with previous and new values of the fields. The values are censored if they are sensitive (for example a password hash), however a hash of the changed value will still be logged.
```

Comment:
Suggestion (and question!): "Sensitive fields will be censored at the API level and replaced with a [what, exactly?] in data sent over the network to be displayed.  A hash of the changed value will still be logged on the server."

Reply:
valid question about what is displayed to user when the value is censored...